app/core/models/tags.py

```diff
@@ -9,3 +18,14 @@ class ReservedTags(enum.IntEnum):
     # can be used for problems with mistakes or for dummies/tests
     PROB_UNGRADED = 3
 
+
+    @staticmethod
```

Comment:
```suggestion
     @classmethod
    def make(cls) -> list[Tag]:
        return [
            Tag(id=cls.MAX_TRIES_UNLIMITED, name="tries unlimited"),
            Tag(id=cls.ANS_CASE_INSENSETIVE, name="answer case insensetive"),
            Tag(id=cls.ANS_DONT_TRIM, name="answer not trimmed"),
            Tag(id=cls.PROB_UNGRADED, name="problem ungraded"),
        ]
```

Как вариант можно так сделать, не придется костыльно T задавать